packages/z3/z3.4.14.1/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+maintainer: "weng@cs.jhu.edu"
+authors: "MSR"
+homepage: "https://github.com/Z3prover/z3"
+bug-reports: "https://github.com/Z3prover/z3/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Z3prover/z3.git"
+depexts: [
+  ["python3-distutils"] {os-family = "debian"}
```

Comment:
This is all right. If it starts growing because many other distributions start requiring it then we would suggest to create a separate conf- package for it